src/views/testUtils.js

```diff
@@ -17,7 +17,7 @@ export const visit = path => {
     // From the docs: Note that, even though goto normally waits
     // for all the resources on a page to load, a timeout exception
     // is only raised if the DOM itself has not yet loaded.
-    gotoTimeout: 4000
+    gotoTimeout: process.env.TEST_TIMEOUT || 4000
```

Comment:
Hmm, I'll see what I can do. Not sure how to test that the changes work though. I have gotten timeouts a couple times, but it's sporadic for me. Generally, running again gets passed the timeouts for me.